lib/google_product_change.dart

```diff
@@ -0,0 +1,35 @@
+class GoogleProductChangeInfo {
```

Comment:
This file should also be added to `object_wrappers.dart`

Reply:
I exported these in `purchases_flutter.dart` but I'll move it to `object_wrappers.dart` 😁 

```dart
export 'google_product_change.dart';
export 'object_wrappers.dart';
export 'upgrade_info.dart';
```

Reply:
oh I see, yeah thanks for moving those over there